src/DMXSerial2.cpp

```diff
@@ -805,6 +814,78 @@ void DMXSerialClass2::_processRDMMessage(byte CmdClass, uint16_t Parameter, bool
 
 // ADD: PARAMETER_DESCRIPTION
 
+    } else if (Parameter == SWAPINT(E120_SENSOR_DEFINITION)) { // 0x0200
+      if (CmdClass == E120_GET_COMMAND) {
+        if (_rdm.packet.DataLength != 1) {
+          // Unexpected data
+          nackReason = E120_NR_FORMAT_ERROR;
+        } else if (_rdm.packet.SubDev != 0) {
+          // No sub-devices supported
+          nackReason = E120_NR_SUB_DEVICE_OUT_OF_RANGE;
+        } else {
+          uint8_t sensorNr = _rdm.packet.Data[0];
+          if (sensorNr >= _initData->sensorsLength) {
```

Comment:
Actually, double-checking the standard, it says "Valid sensor numbers are in the range from 0x00 – 0xFE. The sensor number 0xFF is used to address all sensors." and "When a device or sub-device is fitted with a single sensor, it would return a value of 0x01 for the Sensor Count. This sensor would then be addressed as Sensor Number 0x00 when using the other sensor-related parameter messages.".

So you can have 255 sensors, which are addressed as 0 to 254.